buildspec.yml

```diff
@@ -0,0 +1,135 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: '1.3.0'
+    CPU_FRAMEWORK_BINARY: 'https://files.pythonhosted.org/packages/71/64/49c5125befd5e0f0e17f115d55cb78080adacbead9d19f253afd0157656a/mxnet-1.3.0.post0-py2.py3-none-manylinux1_x86_64.whl'
+    CPU_PY_VERSION: '2'
```

Comment:
what would happen to build duration if we tested all 4 images (cpu, gpu) x (py2, py3)?

Reply:
The test times should be just doubled. So I continue to use what we did in TC, py3 for gpu and py2 for cpu.

Reply:
ok